apps/prepopulate/data_init/vocabularies.json

```diff
@@ -168,7 +168,8 @@
                     {"name": "NINJS", "qcode": "ninjs"},
                     {"name": "NewsML G2", "qcode": "newsmlg2"},
                     {"name": "Email", "qcode": "email"},
-                    {"name": "Newsroom NINJS", "qcode": "newsroom ninjs"}
+                    {"name": "Newsroom NINJS", "qcode": "newsroom ninjs"},
+                    {"name": "IDML", "qcode": "idml"}
```

Comment:
`"Adobe InDesign"` sound like a better name :)

Reply:
@tomaskikutis , yes, it sounds more familiar for a users.  Adobe indesign can also be a *.indd file. In this case we can name it as Adobe InDesign (IDML).